docs/src/main/paradox/client.md

```diff
@@ -165,6 +165,11 @@ avoid leaking in the latter case, you should call `.close()` on the client.
 When the connection breaks, the client will start failing requests and try reconnecting
 to the server automatically.
 
+If a connection can not be established after the configured number of attempts then
+the client closes its self. When this happens the @scala[`Future`]@java[`CompletionStage`] 
+returned by `closed()` will complete with a failure. You do not need to call `close()` in
+this case. 
```

Comment:
This means creating a client and not listening on the `closed()` future is basically a bug, right? Do we have examples we have to update to show that practice?

Reply:
Yeah that also makes me think this should be a much larger default to take advantage of the built in backoff etc